Sources/StreamChatUI/ChatChannel/ChatChannelVC+SwiftUI.swift

```diff
@@ -0,0 +1,22 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import SwiftUI
+
+@available(iOSApplicationExtension, unavailable)
+extension ChatChannelVC: SwiftUIRepresentable {
+    public var content: (
```

Comment:
I would recommend also make a single line for two-argument declarations/function/etc